src/lib/PhpCsFixer/Config.php

```diff
@@ -0,0 +1,213 @@
+            'protected_to_private' => true,
+            'return_type_declaration' => true,
+            'semicolon_after_instruction' => true,
+            'short_scalar_cast' => true,
+            'single_blank_line_before_namespace' => true,
+            'single_class_element_per_statement' => true,
+            'single_line_comment_style' => [
+                'comment_types' => ['hash'],
+            ],
+            'single_quote' => true,
+            'single_trait_insert_per_statement' => true,
+            'space_after_semicolon' => [
+                'remove_in_empty_for_expressions' => true,
+            ],
+            'standardize_increment' => true,
+            'standardize_not_equals' => true,
+            'ternary_operator_spaces' => true,
+            'trailing_comma_in_multiline_array' => true,
+            'trim_array_spaces' => true,
+            'unary_operator_spaces' => true,
+            'whitespace_after_comma_in_array' => true,
+            'yoda_style' => true,
+
+            'dir_constant' => true,
+            'ereg_to_preg' => true,
+            'error_suppression' => true,
+            'fopen_flag_order' => true,
+            'fopen_flags' => ['b_mode' => false],
+            'function_to_constant' => true,
+            'implode_call' => true,
+            'is_null' => true,
+            'modernize_types_casting' => true,
+            'native_constant_invocation' => [
+                'fix_built_in' => false,
+                'include' => [
+                    'DIRECTORY_SEPARATOR',
+                    'PHP_SAPI',
+                    'PHP_VERSION_ID',
+                ],
+                'scope' => 'namespaced',
+            ],
+            'native_function_invocation' => [
+                'include' => [NativeFunctionInvocationFixer::SET_COMPILER_OPTIMIZED],
+                'scope' => 'namespaced',
+                'strict' => true,
+            ],
+            'no_alias_functions' => true,
+            'no_homoglyph_names' => true,
+            'non_printable_character' => true,
+            'php_unit_construct' => true,
+            'php_unit_mock_short_will_return' => true,
+            'psr4' => true,
+            'self_accessor' => true,
+            'set_type_to_cast' => true,
+            'concat_space' => ['spacing' => 'one'],
+            'array_syntax' => ['syntax' => 'short'],
+            'simplified_null_return' => false,
+            'phpdoc_align' => false,
+            'phpdoc_to_comment' => false,
+            'cast_spaces' => false,
+            'blank_line_after_opening_tag' => false,
+            'single_blank_line_before_namespace' => false,
+            'space_after_semicolon' => false,
+            'header_comment' => [
+                'commentType' => 'PHPDoc',
+                'header' => static::EZPLATFORM_PHP_HEADER,
+                'location' => 'after_open',
+                'separate' => 'top',
+            ],
+            'yoda_style' => false,
```

Comment:
was this not loosely enforced in adminUI?

Reply:
I think so but since it was set to `false`, enabling it right now will force us to do refactorings which is something I do not want to do (all maintained branches * repos).